pom.xml

```diff
@@ -124,6 +123,22 @@
 					<sourcepath>${basedir}/plugins</sourcepath>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-packaging-plugin</artifactId>
+				<version>${tycho-version}</version>
+				<dependencies>
+					<dependency>
+						<groupId>org.eclipse.tycho.extras</groupId>
+						<artifactId>tycho-buildtimestamp-jgit</artifactId>
+						<version>${tychoExtrasVersion}</version>
+					</dependency>
+				</dependencies>
+				<configuration>
+					<strictBinIncludes>false</strictBinIncludes>
+					<format>'${BUILD_ALIAS}-v'${BUILD_TIMESTAMP}</format>
```

Comment:
why not ${BUILD_ALIAS}.v'${BUILD_TIMESTAMP}. Ie eclipse.core is using dot instead of dash